frontend/public/components/events.jsx

```diff
@@ -281,7 +281,7 @@ class EventStream extends SafetyFirst {
       })
       .onclose(evt => {
         if (evt && evt.wasClean === false) {
```

Comment:
We might remove the `wasClean` check (although presumably it will always be false)

Reply:
Yes, in my limited testing `wasClean` is always false.

Screenshot:
![image](https://user-images.githubusercontent.com/12733153/51686519-84933d00-1fbe-11e9-8382-ec78a8908f55.png)